react/week1/todolist/src/App.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import Activities from "./activities";
+import {TodoList} from './TodoList';
```

Comment:
As a minor adjustments take a look at naming conventions and whatever you pick for file names be consistent but otherwise great!